pkg/agent/apiserver/apiserver.go

```diff
@@ -90,7 +91,11 @@ func installHandlers(aq agentquerier.AgentQuerier, npq querier.AgentNetworkPolic
 func installAPIGroup(s *genericapiserver.GenericAPIServer, aq agentquerier.AgentQuerier, npq querier.AgentNetworkPolicyInfoQuerier, v4Enabled, v6Enabled bool) error {
 	systemGroup := genericapiserver.NewDefaultAPIGroupInfo(systemv1beta1.GroupName, scheme, metav1.ParameterCodec, codecs)
 	systemStorage := map[string]rest.Storage{}
-	supportBundleStorage := supportbundle.NewAgentStorage(ovsctl.NewClient(aq.GetNodeConfig().OVSBridge), aq, npq, v4Enabled, v6Enabled)
+	iptables, err := iptables.New(v4Enabled, v6Enabled)
+	if err != nil {
+		return fmt.Errorf("failed to new iptables client: %v", err)
```

Comment:
maybe "create" or "instantiate" since "new" is not a verb

Reply:
will update